lstchain/calib/camera/calibrator.py

```diff
@@ -143,14 +141,15 @@ def _calibrate_dl0(self, event, telid):
         create dl0 level, for the moment copy the r1
         """
         waveforms = event.r1.tel[telid].waveform
+        tel_calibration = self.mon_data.tel[telid].calibration
```

Comment:
Hi @yrenier, 
since in this module we play with different  types of calibration data (self and event), I prefer not to introduce aliases in order to keep the formula immediately understandable.

Reply:
I did this for readability of the formulas and for performance optimization (probably negligible) . 
I can revert it or change the variable name to `self_tel_calib` for example. Let me know what you prefer.

Reply:
@yrenier could you please revert it? afterwards, we can merge this PR

Reply:
done

Reply:
@FrancaCassol are you happy now with this PR?